dfd_tests/settings.py

```diff
@@ -19,20 +24,25 @@
 ROOT_URLCONF = 'dfd_tests.urls'
 SITE_ID = 1
 
-INSTALLED_APPS = (
-    'django.contrib.admin',
-    'django.contrib.auth',
-    'django.contrib.contenttypes',
-    'django.contrib.sessions',
-    'django.contrib.messages',
-    'django.contrib.staticfiles',
-    'django.contrib.sites',
-    'dfd_tests',
+cms_apps = [
     'menus',
     'cms',
     'treebeard',
-    'form_designer',
     'form_designer.contrib.cms_plugins.form_designer_form',
+]
+
+INSTALLED_APPS = tuple(
+    [
```

Comment:
Just make it a list. A list like that being a tuple wasn't a good idea in the first place.